tests/test_environ_config.py

```diff
@@ -282,3 +283,41 @@ def bad_formatter(help_dicts):
 
         help_str = environ.generate_help(Parent, formatter=bad_formatter)
         assert help_str == "Not a good formatter"
+
+    def test_frozen(self):
```

Comment:
Please add [test docstrings](https://jml.io/pages/test-docstrings.html) for all tests.

Reply:
Are changes ok?

Reply:
Not entirely our style but I'm not gonna pester you any further. :)